nuget.config

```diff
@@ -3,5 +3,6 @@
 	<packageSources>
 		<add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
 		<add key="MyGet" value="https://dotnet.myget.org/F/uwpcommunitytoolkit/api/v3/index.json" protocolVersion="3" />
-	</packageSources>
+    <add key="MUX-Shared" value="https://pkgs.dev.azure.com/ms/microsoft-ui-xaml/_packaging/MUX-Shared/nuget/v3/index.json" />
```

Comment:
Do we still need this?

Reply:
We **do** still need this. Its where the package is published. It is a public feed.